Gruntfile.js

```diff
@@ -19,9 +19,53 @@ module.exports = function (grunt) {
                 },
                 src: ['test/**/*.test.js']
             }
+        },
+        copy: {
```

Comment:
So you have to have the sample plugin locally for this to work? Should this tackle acquiring it for you?